group/templates/group/groupList.html

```diff
@@ -21,138 +21,57 @@
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.-->
 
+{% load static %} 
+{% block import_source %}
+<link href="{% static 'group/css/group.css' %}" rel="stylesheet">
+{% endblock import_source %} 
 {% extends "index/base.html" %}
```

Comment:
I got ``<ExtendsNode: extends "index/base.html"> must be the first tag in the template.``